packages/tools/devtools/devtools-test-app/src/FluidObject.ts

```diff
@@ -11,6 +11,61 @@ import { SharedMatrix } from "@fluidframework/matrix/internal";
 import { SharedString } from "@fluidframework/sequence/internal";
 import { type ITree, SchemaFactory, TreeViewConfiguration } from "@fluidframework/tree";
 import { SharedTree } from "@fluidframework/tree/internal";
+
+/**
+ * Additional Data Object added to the {@link AppData}.
+ */
+export class AppDataTwo extends DataObject {
+	/**
+	 * Key in the app's `rootMap` under which the SharedString object is stored.
+	 */
+	private readonly sharedTextKey = "shared-text";
+
+	/**
+	 * Key in the app's `rootMapTwo` under which the SharedDirectory object is stored.
+	 */
+	private readonly initialObjectsDirKey = "rootMapTwo";
```

Comment:
This seems to be used as a key for a sub-directory. Might be worth renaming the value to "sub-directory" or something. It's not really a "map", nor is it the "root".

Reply:
Probably also worth renaming the property to something like "subDirectoryKey"

Reply:
Or we could just remove the sub-directory. Either way.